docs/user-guide/src/flake/prow.md

```diff
@@ -0,0 +1,9 @@
+# Prow flakes
+
+## Multiple PRs stuck in merge queue
+
+At times you will find multiple PR have passed all the tests and also have the required labels `approve` and `lgtm`. All the necessary checks have passed but prow cannot merge them. In such cases, it is mostly found that one of the PRs are not rebased properly . You have to put an hold on that PR and the other PR should go in. Once you rebase and push the first PR, that should also go in without any issue. So `/hold` should do the trick and prow should be able to merge the PR one by one.
+
+## Certain PRs not getting merged
+
+PRs which are modifying/adding files in `.github/workflows/` directory do not get merged automatically by Prow. These PRs tend to get stuck inspite of all the tests passed and all the labels present. We have identified that the reason for this is the branch not being presented in `upstream` but in `Nordix`. We did not identify the reason behind it. The solution is to push the branch in `upstream`. Once prow detects the branch is present in `upstream`, it will be able to merge the `Nordix` branch. You don't have to open a PR from `upstream` branch. Only its existense is enough. However only people who have elevated permissions can push branches `upstream` for saftey reasons. So if you face such issue please ask assistance by emailing `estjorvas@est.tech`.
```

Comment:
- presented -> present
- `upstream` but in `Nordix` -> `upstream` (metal3-io) but in `origin` (Nordix, or other fork)
- In general, this change is in metal3-docs which   shouldnt't be ESJ or Nordix specific
- saftey -> safety
- estjorvas@est.tech is private list, it cannot be emailed to by externals. Change it to metal3-dev google groups list.

Reply:
As we checked, `estjorvas@est.tech` is not private and metal3-dev google groups list is not where we should send these emails. So lets keep it as it is.

Reply:
It is private, but there is a setting for private lists to allow sending email to those lists. It is just not visible elsewhere than admin interface. But yeah, we can use it.

Reply:
if you can send email from outside, I dont consider it private.